File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/SearchPreAuthCriteria.cs
using System;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class SearchPreAuthCriteria
    {
        public string PreAuthNumber { get; set; }
        public int PreAuthTypeId { get; set; }
        public int PreAuthStatusId { get; set; }
        public int HealthCareProviderId { get; set; }
        public DateTime? DateAuthorisedFrom { get; set; }
        public DateTime? DateAuthorisedTo { get; set; }
        public int PageSize { get; set; }
        public int PageNumber { get; set; }
        public int? ClaimId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/medicare/src/app/preauth-manager/models/search-preauth-criteria.ts
export class SearchPreAuthCriteria {
    preAuthNumber: string;
    preAuthTypeId: number;
    preAuthStatusId: number;
    healthCareProviderId: number;
    dateAuthorisedFrom: any;
    dateAuthorisedTo: any;
    pageNumber: number | null;
    pageSize: number | null;
    claimId: number | null;
}